macos/xcode-command-line-tools-setup.md

```diff
@@ -13,8 +13,12 @@ xcode-select --install
 This should trigger a dialog box to appear asking if you want to install the
 command line tools, click Ok and wait for the tools to finish installing.
 
-Once this is finished, move on to [Installing homebrew]
+Once this is finished, move on to Installing Homebrew
+
+<div>
```

Comment:
You can just do this with markdown and image links can't you?

Reply:
You can, I was trying to leave it open to use stuff from the browser for an eventual hook up from the ansible file. Hitting that prev button would then take you to the macOS guide or the WIndows or Ubuntu depending on your history. I also had trouble aligning the markdown and image links for the next button to the right of the page.

Reply:
If you think we don't need the prev button once they're in Ansible I can change it back for markdown and image links and then update the other two Operating system guides as well. Just let me know what you would like to happen.